contracts/MemberToken.sol

```diff
@@ -154,6 +154,8 @@ contract MemberToken is ERC1155Supply, Ownable {
         uint256[] memory amounts,
         bytes memory data
     ) internal override {
+        super._beforeTokenTransfer(operator, from, to, ids, amounts, data);
```

Comment:
re: your earlier comment, what your saying is because we don't have this hook, the functions from `ERC1155 supply` aren't able to be used?

Reply:
yes